src/hyperion/external_interaction/callbacks/grid_detection_callback.py

```diff
@@ -1,15 +1,32 @@
+from typing import TypedDict
+
 import numpy as np
 from bluesky.callbacks import CallbackBase
-from dodal.devices.fast_grid_scan import GridScanParams
 from dodal.devices.oav.oav_detector import OAVConfigParams
-from dodal.devices.panda_fast_grid_scan import PandAGridScanParams
 from event_model.documents import Event
 
 from hyperion.device_setup_plans.setup_oav import calculate_x_y_z_of_pixel
 from hyperion.log import LOGGER
 from hyperion.parameters.constants import CONST
 
 
+class GridParamUpdate(TypedDict):
+    transmission_frac: float
+    exposure_time_s: float
+    x_start_um: float
+    y_start_um: float
+    y2_start_um: float
+    z_start_um: float
+    z2_start_um: float
+    x_steps: int
+    y_steps: int
+    z_steps: int
+    x_step_size_um: float
+    y_step_size_um: float
+    z_step_size_um: float
+    set_stub_offsets: bool
+
+
 class GridDetectionCallback(CallbackBase):
```

Comment:
Should: I don't think `run_up_distance_mm` is ever used here now so we can remove